src/index.html

```diff
@@ -7,8 +7,34 @@
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <a href="#" class="logo">
+        <img src="images/logo.png" alt="logo">
+      </a>
+      <nav class="nav">
+        <ul class="nav-list">
+          <li><a href="#" class="header-link is-active">Apple</a></li>
```

Comment:
```suggestion
    <header class="header">
      <a href="#" class="logo">
        <img src="images/logo.png" alt="logo">
      </a>
      
      <nav class="nav">
        <ul class="nav-list">
          <li>
            <a href="#" class="header-link is-active">
              Apple
            </a>
          </li>
```
pay attention to the code style